features/step_definitions/logging.rb

```diff
@@ -180,11 +180,14 @@
     | component=kibana |
   })
   cl.wait_until_kibana_is_ready
-  # lastly check the cronjob.
-  if env.version_cmp('4.5', user: user) < 0
+  # check the cronjob curator.
+  if env.version_cmp('5.0', user: user) < 0
     raise "Failed to find cronjob for curator" if cron_job('curator').schedule.nil?
-  else
-    cj_names = ["elasticsearch-delete-app", "elasticsearch-delete-infra", "elasticsearch-rollover-app", "elasticsearch-rollover-infra"]
+  end
+
+  # check the cronjob index management
+  if env.version_cmp('4.4', user: user) > 0
+    cj_names = ["elasticsearch-im-app", "elasticsearch-im-audit", "elasticsearch-im-infra"]
     for cj_name in cj_names do
       raise "Failed to find cronjob for #{cj_name}" if cron_job(cj_name).schedule.nil?
```

Comment:
How about `raise "Failed to find cronjob for #{cj_name}" unless cron_job(cj_name).exists? ` ?